packages/fxa-auth-server/lib/senders/emails/templates/passwordChangeRequired/index.mjml

```diff
@@ -0,0 +1,36 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-include path="./css/global.css" type="css" css-inline="inline" />
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="passwordChangeRequired-title">Password Change Required</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-body align-left">
+      <span data-l10n-id="passwordChangeRequired-suspicious-activity">We detected suspicious behavior on your Firefox account. To prevent unauthorized access to your Firefox Account, we’ve disconnected all devices on your account and are requiring you to change your password as a precaution.</span>
+    </mj-text>
+
+    <mj-text css-class="text-body align-left">
+      <span data-l10n-id="passwordChangeRequired-sign-in">Sign back into any device or service where you use your Firefox account and follow the steps that will be presented to you.</span>
+    </mj-text>
+
+    <mj-text css-class="text-body align-left">
+      <span data-l10n-id="passwordChangeRequired-different-password"><b>Important:</b> Pick a different password than what you were previously using and make sure that it is different from your email account.</span>
+    </mj-text>
+
+    <mj-text css-class="text-body align-left lg-bottom-margin">
+      <span data-l10n-id="passwordChangeRequired-signoff">
+        Best,<br />
+        The Firefox Accounts Team
+      </span>
+    </mj-text>
+  </mj-column>
+</mj-section>
```

Comment:
This template is going to look fairly different because the new templates introduce a consistent line height for paragraphs:

<img width="1777" alt="Screen Shot 2021-09-15 at 12 51 17 PM" src="https://user-images.githubusercontent.com/6392049/133486223-046e137b-1f58-4c65-b000-f8584ee44e97.png">

Reply:
Looks way nicer in the new template IMO.